.github/workflows/codeql.yml

```diff
@@ -29,9 +29,9 @@ jobs:
           - os: ubuntu-latest
             pool_scalable: 'ON'
             # Windows doesn't recognize 'CMAKE_BUILD_TYPE', it uses '--config' param in build command
-            extra_build_option: ['-DCMAKE_BUILD_TYPE=Release']
+            extra_build_option: '-DCMAKE_BUILD_TYPE=Release'
           - os: windows-latest
-            pool_scalable: 'OFF'
+            pool_scalable: 'ON'
```

Comment:
just enable it in cmake options and remove this from matrix

Reply:
Done